samples/drivers/led_ws2812/boards/yd_esp32.overlay

```diff
@@ -0,0 +1,38 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/dt-bindings/led/led.h>
+
+&pinctrl {
+	spim2_default: spim2_default {
+		group2 {
+			pinmux = <SPIM2_MOSI_GPIO16>;
+			output-low;
+		};
+	};
+};
+
+&spi2 {
```

Comment:
I agree that this should be moved into board's default .dts file instead as part of the HW.

Reply:
Done. This was moved into board's default `.dts` file:

https://github.com/zephyrproject-rtos/zephyr/blob/6a40fd4972c200ea6f12f294e8b302012418570d/boards/xtensa/yd_esp32/yd_esp32.dts#L109-L131

https://github.com/zephyrproject-rtos/zephyr/blob/6a40fd4972c200ea6f12f294e8b302012418570d/boards/xtensa/yd_esp32/yd_esp32.dts#L55-L62

---

Cc: @kartben, @henrikbrixandersen